core/vm/evm.go

```diff
@@ -686,7 +685,7 @@ func (evm *EVM) CreateETX(toAddr common.Address, fromAddr common.Address, gas ui
 		return []byte{}, 0, 0, fmt.Errorf("CreateETX overflow error: too many ETXs in cache")
 	}
 	// create external transaction
-	etxInner := types.ExternalTx{Value: value, To: &toAddr, Sender: fromAddr, OriginatingTxHash: evm.Hash, ETXIndex: uint16(index), Gas: gas, Data: data, AccessList: evm.AccessList}
+	etxInner := types.ExternalTx{Value: value, To: &toAddr, Sender: fromAddr, EtxType: types.DefaultType, OriginatingTxHash: evm.Hash, ETXIndex: uint16(index), Gas: gas, Data: data, AccessList: evm.AccessList}
```

Comment:
Need to set ConversionType if etx is a conversion

Reply:
yeah changed it to have if conversion is true